# Question:

### *How many men versus woman survived the Titanic?*

### Study:
I will answer this using a census study because all of the people that were on the Titanic are in the data

In [53]:
import pandas as pd

In [54]:
df = pd.read_csv(r"C:\Users\*******\!jupyter\data\titanic.csv") # if you're looking at this line from GitHub, I had to hardcode it cause it's the only way that would work

In [55]:
# I could also do:
# survivors_by_sex = df[df["Survived"] == 1].groupby("Sex").size()
# but nah standard Python is more fun in my opinion

m_survivors = 0
f_survivors = 0

for i in range(len(df)):
    if str(df.Sex[i]).lower() == "male" and int(df.Survived[i]) == 1:
        m_survivors += 1
    elif str(df.Sex[i]).lower() == "female" and int(df.Survived[i]) == 1:
        f_survivors += 1
    else:
        continue


print(f"m: {m_survivors} | f: {f_survivors}")

m: 109 | f: 233


### Conclusion:

According to my analsys, more woman *(233)* survived than men *(109)*. This is most likely because when the ship was sinking, they would prioritize saving woman and children rather than adult men.